karabo_data/geometry2.py

```diff
@@ -216,13 +221,15 @@ def from_quad_positions(cls, quad_pos, asic_gap=2, panel_gap=29):
             x_orient = quads_x_orientation[quad]
             y_orient = quads_y_orientation[quad]
             p_in_quad = p % 4
-            corner_y = quad_corner[1] - (p_in_quad * (128 + panel_gap))
+            corner_y = (quad_corner[1] * px_conversion)\
+                       - (p_in_quad * (128 + panel_gap))
 
             tiles = []
             modules.append(tiles)
 
             for a in range(8):
-                corner_x = quad_corner[0] + x_orient * (64 + asic_gap) * a
+                corner_x = (quad_corner[0] * px_conversion)\
+                           + x_orient * (64 + asic_gap) * a
```

Comment:
Can you remind me again where the 64 comes from?

Reply:
That's the ASIC/tile size in pixels (this bit of the code is for AGIPD). There's a named value that it should be using here; I'll fix that.